docs/learn.md

```diff
@@ -3,20 +3,20 @@ title: Learn ES2015
 id: learn
 ---
 
-<blockquote class="alert alert--info">
+:::info
```

Comment:
Prettier currently do not support admonitions: https://github.com/prettier/prettier/issues/8640 Fortunately we are not affected yet because we are not using the `proseWrap` option.